src/components/Input.jsx

```diff
@@ -60,6 +62,14 @@ const Input = forwardRef(
 
     const handleChange = rejectCharsRegex ? handleRegexChange : onChange;
 
+    const handleOnBlur = e => {
+      if (!disableTrimOnBlur) {
+        e.target.value = value.trim();
+        handleChange(e);
```

Comment:
What will happen when the Input is used as an uncontrolled component?

Reply:
The value is taken from an internal state, ref: https://github.com/bigbinary/neeto-ui/pull/1906/files#diff-3e66527f9b6fc3f34b0bb85e7c327daa7f9f080f1183c480ed69e4c13a98bc64R40-R46

Reply:
For uncontrolled, the line just above `e.target.value = value.trim();` will update the value.